_benchmark/commands/execute-test.md

```diff
@@ -0,0 +1,172 @@
+- `--runtime-jdk`: The major version of JDK to use.
+- `--client-options`: Defines a comma-separated lists of clients to use. All options are passed to the OpenSearch Python client. Default is `timeout:60`.
+
+### Cluster
+
+The following options set which cluster options. 
+
+- `--target-hosts`: Defines a comma-separated list of host:port pairs which should be targeted if using the pipeline `benchmark-only`. Default is `localhost:9200`.
+- `--load-worker-coordinator-hosts`: Defines a comma-separated list of hosts that coordinate loads. Default is `localhost`.
+- `--enable-worker-coordinator-profiling`: Enables an analysis of the performance of call in OpenSearch Benchmark's worker coordinator. Default is `false`.
+
+### Provisioning
+
+The following options help customize how OpenSearch Benchmark provisions OpenSearch and workloads.
+
+- `--provision-config-repository`: Defines the repository from which OpenSearch Benchmark loads `provision-configs` and `provision-config-instances`. 
+- `--provision-config-path`: Defines the path to the `--provision-config-instance` and any OpenSearch plugin configurations to use.
+- `--provision-config-revision`: Defines a specific Git revision in the `provision-config` that OpenSearch Benchmark should use.
+- `--provision-config-instance`: Defines the `--provision-config-instance` to use. You can see possible configuration instances using the command `opensearch-benchmark list provision-config-instances`. 
+- `--provision-config-instance-params`: A comma-separated list of key:value pairs injected verbatim as variables for the `provision-config-instance`.
+
+
+### Workload
+
+The following options determine which workload is used to run the test.
+
+- `--workload-repository`: Defines the repository from where OpenSearch Benchmark loads workloads.
+- `--workload-path`: Defines the path to a downloaded or custom workload.
+- `--workload-revision`: Defines a specific revision from the workload source tree that OpenSearch Benchmark should use.
+- `--workload`: Defines the workload to use based on the workload's name. You can find a list of preloaded workloads using `opensearch-benchmark list workloads`.
+
+### Test procedures
+
+The following options define what test procedures the test uses and which tasks are contained inside the procedure.
+
+- `--test-execution-id`: Defines a unique ID for this test run.
+- `--test-procedure`: Defines a test procedure to use. You can find a list of test procedures using `opensearch-benchmark list test-procedures`.
+- `--include-tasks`: Defines a comma-separated list of test procedure tasks to run. By default, all tasks listed in a test procedure array are run.
+- `--exclude-tasks`: Defines a comma-separated list of test procedure tasks not to run.
+- `--enable-assertions`: Enables assertion checks for tasks. Default is `false`.
+
+### Pipelines
+
+The `--pipeline` option selects a pipeline to run.
```

Comment:
Is there a command to list the type of pipelines as well?
If yes, can we add that command here?